lib/CXGN/Trial/TrialCreate.pm

```diff
@@ -259,9 +266,9 @@ sub save_trial {
 		};
 	}
 	if ($error) {
-		return ( error => $error );
+		return { error => $error };
 	}
-	return ( trial_id => $project->project_id );
+	return { trial_id => $project->project_id };
```

Comment:
returning a different data structure has effects on code that uses this, so the tests don't pass anymore

Reply:
true, thanks for checking. Latest commit should fix this.